Makefile

```diff
@@ -3,38 +3,61 @@ export GO15VENDOREXPERIMENT=1
 SHORT_NAME := deis-e2e
 
 SRC_PATH := /go/src/github.com/deis/workflow-e2e
-DEV_IMG := quay.io/deis/go-dev:0.9.1
-
-RUN_CMD := docker run --rm -e DEIS_ROUTER_SERVICE_HOST=${DEIS_ROUTER_SERVICE_HOST} -e DEIS_ROUTER_SERVICE_PORT=${DEIS_ROUTER_SERVICE_PORT} -v ${CURDIR}:${SRC_PATH} -w ${SRC_PATH} ${DEV_IMG}
-DEV_CMD := docker run --rm -e GO15VENDOREXPERIMENT=1 -v ${CURDIR}:${SRC_PATH} -w ${SRC_PATH} ${DEV_IMG}
-
-TEST_OPTS := -slowSpecThreshold=120.00 -noisyPendings=false
-PARALLEL_TEST_OPTS := ${TEST_OPTS} -p
 
 MUTABLE_VERSION ?= canary
 VERSION ?= git-$(shell git rev-parse --short HEAD)
 
+GINKGO_NODES ?= 15
+
+TEST_OPTS := -slowSpecThreshold=120.00 -noisyPendings=false
+PARALLEL_TEST_OPTS := ${TEST_OPTS} -nodes=${GINKGO_NODES}
+
 DEIS_REGISTRY ?= quay.io/
 IMAGE_PREFIX ?= deis
 IMAGE := ${DEIS_REGISTRY}${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
 MUTABLE_IMAGE := ${DEIS_REGISTRY}${IMAGE_PREFIX}/${SHORT_NAME}:${MUTABLE_VERSION}
 
+JUNIT ?= true
+
+DEV_IMG := quay.io/deis/go-dev:0.9.0
+DEV_CMD_ARGS := --rm -v ${CURDIR}:${SRC_PATH} -w ${SRC_PATH} ${DEV_IMG}
+DEV_CMD := docker run ${DEV_CMD_ARGS}
+DEV_CMD_INT := docker run -it ${DEV_CMD_ARGS}
+RUN_CMD := docker run --rm -e DEIS_CONTROLLER_URL=${DEIS_CONTROLLER_URL} \
+													 -e DEFAULT_EVENTUALLY_TIMEOUT=${DEFAULT_EVENTUALLY_TIMEOUT} \
+													 -e MAX_EVENTUALLY_TIMEOUT=${MAX_EVENTUALLY_TIMEOUT} \
+													 -e JUNIT=${JUNIT} \
+													 -e DEBUG=${DEBUG} \
+													 -v ${HOME}/.kube:/root/.kube \
+													 -v ${CURDIR}:${SRC_PATH} -w ${SRC_PATH} ${IMAGE}
+
+.PHONY: check-controller-url
```

Comment:
I'd be happy to move them.  They are where they are now because that's where they were when I started.